html-intro/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML Intro</title>
+  </head>
+  <body>
+    <img
+      src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/393.png"
+      alt="Blue, penguin pokemon"
+      title="Piplup, the penguin pokemon." />
+    <h1><strong>Piplup</strong></h1>
```

Comment:
`h1` elements will be bold by default, and will already have an emphasis, so no need to put `strong` inside of here